optional_plugins/varianter_yaml_to_mux/avocado_varianter_yaml_to_mux/mux.py

```diff
@@ -101,7 +101,10 @@ def iter_variants(self):
                 pools.append([pool])
         variants = itertools.product(*pools)
         while True:
-            yield list(itertools.chain(*next(variants)))
+            try:
+                yield list(itertools.chain(*next(variants)))
+            except StopIteration:
```

Comment:
I see, py2 does not support the `yield from` shouldn't this be treated by `six.next`? I mean have you tried using

```python
yield list(itertools.chain(*next(variants)))
return
```

(Not that this would be wrong, I'm just curious)